### DISCUSSION TITLE
Add relation between issues

### DISCUSSION THREAD

**Discussion by Gotterbild** [community/community#4928] [Projects and Issues] [answered]
It is often case when one issue blocks another. It will be great to have ability to explicitly set this relation.

Right now in the sidebar there is `Linked Pull Requests`. There can be additional `Related issues`, and you could mark any issue in this list as blocked by current issue or as a blocking for current issue.

**Comment by mattrosno** [comment]
I'm also looking for this. Not so much issue dependency, but using issues for both tasks that get completed, and to track user needs over time.

For example, let's say you use a repository to track job stories, jobs to be done, or any flavor of user outcomes that you continuously address. You also have a repository with code and normal issues that get completed per release, sprint, task, etc.

I'm wanting issues to be related to other issues as a named relationship. For example, development tasks are issues that have a custom relation property named "Addresses" where you are able to select one or many job story issues.

In the new table view, you could see which user outcomes are being addressed in each development task. Or you could have a project view with the inverse so you could primarily view the job story issues and which development tasks are addressing each job story.

Custom relations are what make Notion and Airtable so powerful, with the downside of them being separate system of records from where the work is getting done - GitHub.

Unless I'm missing something, per the [marketing page](https://github.com/features/issues), how are relationships getting tracked? Or is it just currently issues referencing other issues in their timelines? If so, that's great as an audit trail, but not so great for product planning / management.

![image](https://user-images.githubusercontent.com/1691245/136815057-624be585-7a0e-41de-a285-eb78d979d287.png)

**Comment by nebbles** [comment]
I came here to either upvote or request this myself. We currently use [Shortcut (formerly Clubhouse)](shortcut.com) for the bulk of our ticket management, but I would love to be able to migrate the team at some point in the future.

Shortcut has the ability to add related issues and designate a particular type in the relation (related/blocking/blocked by/duplicates/duplicated by).

All related issues are then grouped into a neat list for reference (similar to linked PRs in GitHub's side bar)

<img width="50%" src="https://user-images.githubusercontent.com/12599555/144724933-fb0cb364-17d1-422b-befa-c21a09f1d8e2.png" />

**Comment by coot** [comment]
An idea to consider would be to add a field type (next to `Text`, `SingleSelect`, etc) which allows to list multiple issues or PRs.  The meaning of it would be left to the user, e.g. what kind of relation is it.

**Comment by tgmpje** [comment]
As a workaround, you can snooze an issue by simply adding a comment:
```
@SnoozeThis <url of issue to block on>
```
SnoozeThis will then add a comment as soon as the other issue has been closed. If you install the [app](https://github.com/apps/snoozethis) you can do more advanced stuff, like (re)assigning, closing, adding labels, etc.

**Comment by sophec** [comment]
This is a key missing feature for me as well. Since projects can span multiple repositories, one should also be able to mark an issue as dependent on an issue in another repository.

**Reply by macnicnac** [reply]
Exactly also looking for this!

**Comment by samhld** [comment]
I'm also looking for this!

**Comment by omerbensaadon** [comment]
Hey folks! We're working on a more robust implementation of precisely this! 

For now, you can use Task lists for this functionality: 

https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists

**Reply by Gotterbild** [reply]
Sounds great! Is there any estimation of a deadline? (Maybe I ask a lot 🙂)

**Reply by omerbensaadon** [reply]
There is not, but I can tell you its actively being worked as we speak

**Reply by Gotterbild** [reply]
@omerbensaadon 
Just in case, I'll mention that this should work between repositories or it will be useless for our team.

For example, the `Tracked in` feature of task lists doesn't work between repositories and we have to set that relation manually via comments and issue description.

**Reply by omerbensaadon** [reply]
@Gotterbild totally hear you 😉

**Reply by michael-hawker** [reply]
I think that may cover some scenarios, but not all. For instance, I may want to link an issue specifically to its "Parent" issue.

So being able to have a custom field type of 'issue' for instance and let me select/link to an issue as a value of that field would be great for that.

But it may also just be a list of other random PRs/issues/discussions that I want to track as 'related'. A task list may cover that but has to live in the issue somewhere vs. just being part of the project board the team uses.

**Reply by JavierSegoviaCordoba** [reply]
@omerbensaadon any update?

**Reply by Gary-H9** [reply]
👋  @omerbensaadon same question as above - is there any update? Even if this is no longer being looked at it would be good to know that's the case. 

Echoing the above I think this would offer real value. This is something our product/delivery managers expect if they migrate from JIRA or ZenHub.

**Reply by danielgomezrico** [reply]
This is a blocked beta that is not accepting new users, so maybe this will not fix the requirement

**Reply by maliberty** [reply]
@omerbensaadon can you give an update on the plans beyond task lists?  The "For now" is 1.5 years old

**Comment by robopzet** [comment]
As far as I understand task lists do not allow the definition of specific relationships, they are just links in the text.
Still waiting for a way to define dependencies between issues and view them, e.g. in a tree or gantt chart, or at least an ordered list.

**Comment by michael-hawker** [comment]
I generally find myself going to two separate issues and having them reference each other to get the nice link to appear in the feed of each:
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/24302614/4a99f924-c203-470a-b667-d53f2dd8e823)

This doesn't scale well beyond linking two issues together. Especially, as it's one sided, if someone just mentions an issue in their comment like this:

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/24302614/5764a2cb-4434-4c01-bb37-e833a53a67b8)

It only gets visibility more on the feed of the other linked issue:

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/24302614/f8c720bf-f219-4e98-8f32-24a3745a66bb)

It'd be really nice for the side panel to have a complete list of issues that are _referencing this one_ and that _this one references_:
![image](https://github.com/microsoft/microsoft-ui-xaml/assets/24302614/727f2f76-796d-4373-8110-91cb29e24968)

For the above example, [from this issue](https://github.com/microsoft/WindowsAppSDK/issues/3546), I would just have a summary and see a list of related issues something like:
- https://github.com/microsoft/microsoft-ui-xaml/issues/6452 (currently buried in a comment)
- https://github.com/microsoft/microsoft-ui-xaml/issues/7830 (down at the bottom of the issue in the feed)

**Comment by dachosen1** [comment]
I'm also looking for this feature. Is the GitHub product team monitoring this discussion? It seems that this issue has been outstanding for quite some time, yet the feature still doesn't exist.

**Reply by joeyfigaro** [reply]
Issue dependencies have been [requested since 2014](https://github.com/isaacs/github/issues/210) with no real movement, unfortunately.

**Comment by JyotiBB** [comment]
It seems interesting to learn that the Projects and Issues team might be considering adding a dependency to an issue.

**Comment by harrisonhoward** [comment]
Issue relationships are important for projects of any size. If I have an issue that can't be completed due to a dependency on a different issue. I have no way of linking those together other than "mentioning" it in the comments.

Tracking these isn't possible. I think a good example of a good way to implement an issue tracker is JIRA, the platform is more advanced but the simplicity of linking issues is something GitHub would benefit from, especially for smaller projects. The growth of the projects is being held back by this very simple feature.

**Comment by joeyfigaro** [comment]
If the product team needs inspiration on how this could/might work, [Linear](https://linear.app/) does a beautiful job of managing relationships between tickets.

**Reply by AleixMT** [reply]
Thank you for the Linear recommendation, if you use the GitHub integration I think it does the job. I also think that the team could use some inspiration from GitLab, since this feature is completely implemented there:

![image](https://github.com/user-attachments/assets/647765ca-4cb6-42c7-9de1-04b65e523cd7)

**Comment by beedell-roke** [comment]
GitLab supports this, as https://gitlab.com/gitlab-org/gitlab/-/blob/c03c1be71de6f26f19862ee203084717dbf4c792/doc/user/project/issues/related_issues.md#add-a-linked-issue explains. *I* suggest that you move there. I have.

**Comment by kelly-sovacool** [comment]
Trello does this very well, it would be nice to have similar functionality for GitHub Projects with the roadmap view.

**Comment by AleixMT** [comment]
I have been waiting for this feature [for years](https://github.com/isaacs/github/issues/210)... 

It feels annoying that we can track all types of things in GitHub but we can't define dependency relations between issues. Do I really need to move all my project organization to Trello for a bunch of arrows? Please, add this feature to GitHub projects ASAP if you are so kind.

**Reply by beedell-roke** [reply]
> https://github.com/orgs/community/discussions/4928#discussioncomment-10222679

@AleixMT, why Trello specifically, rather than GitLab, per https://github.com/orgs/community/discussions/4928#discussioncomment-9881307? I'm not aware of Trello providing source control.

**Reply by AleixMT** [reply]
It is true @RokeJulianLockhart that GitLab does a better job than GitHub at managing the projects plus it has source control. That is definitely the best option if you just started using GitHub.

The thing is that probably not everyone wants to move all of his repos into GitLab only because the lack of this feature in GitHub. We can keep using GitHub for everything else and then use a third tool such as Trello to manage the issues of the project. Due to reading this discussion I also discovered Linear, which seems very good at managing the project and being integrated with GitHub.

**Comment by falvius** [comment]
+1

In my opinion, GitHub projects would benefit a lot from these relations links.

Especially because these are basic features for other project tools (like Redmine, Codebeamer, Polarion, Jira, etc..).

New fields like these should be possible and are high in value: 
- issue A is a child or parent from issue B
- issue A is related to issue B (...in some way, but for starters "just related" would for me suffice)

**Comment by loeffel-io** [comment]
Please! 🙏

**Comment by ghost** [comment]
ended up here while looking for the functionality, would be great to have this feature

**Comment by azenMatt** [comment]
Hi everyone - we currently have a beta in progress for this! You can see more info here: 

- #131957 

Access to that beta also give you access to: 

- #112806 
- #132506 

If you would like to join this beta - please let me know your organization names below and I can add you in. I will add a reaction to your comment once I have added you. Thanks! and please share your feedback once you have tried it!

**Reply by loeffel-io** [reply]
Please add the github.com/mindful-hq organization to the beta! ❤️

**Reply by matmair** [reply]
Please add github.com/inventree and github.com/invenhost organizations to the beta - very excited for this! Tasklists was already a great improvement, this sounds even better.

**Reply by beedell-roke** [reply]
> https://github.com/orgs/community/discussions/4928#discussioncomment-10612280

@azenMatt, can individuals gain access?

**Reply by vasil-nesterov** [reply]
@azenMatt Please add http://github.com/fresh811 to the beta. Cheers!

**Reply by agilesteel** [reply]
Please add the https://github.com/formslogic organization to the beta. Thank you!

**Reply by Norbiros** [reply]
Please add https://github.com/CrabCraftDev to beta. Thanks!

**Reply by innershell** [reply]
Please add https://github.com/innershell to beta. Thanks! This would really help me with creating a requirements traceability matrix for my regulated project.

**Reply by mssngr** [reply]
Org is `collegeboard-software`. Is it possible to limit adding us to the Beta only with a single Project? (The "Hummingbird" Project.)

**Reply by azenMatt** [reply]
right now the beta is only for orgs, 

@mssngr - it does have to be for the full org. I haven't added your org but let me know if you would like me to

**Reply by meedsharif** [reply]
Please add [https://github.com/BrownBrickio](https://github.com/BrownBrickio) to the beta. Thanks!

**Reply by leaf-johannes** [reply]
We would like this beta https://github.com/Leaf-Software-Studio 🙏

**Reply by dminchev** [reply]
@azenMatt we would like to try this sub issues for https://github.com/wogiApps. Thanks!

**Reply by Rawdreen** [reply]
I'm really impressed with the direction GitHub Projects is heading and would love to try it out beta releases. 
I'd be happy to share my feedback after testing it. Thanks! 

https://github.com/Rawdreen

**Reply by simensol** [reply]
Please add https://github.com/eisi-no to the beta, thanks!

**Reply by ptsouchlos** [reply]
Requesting beta access for @xstrahl

**Reply by binzcodes** [reply]
Thank you @azenMatt I'd love beta access to test this: https://github.com/binzcodes

**Reply by navasvarela** [reply]
Thank you @azenMatt , please give access to the beta for https://github.com/NillionNetwork

**Reply by azenMatt** [reply]
We now have an automated way for you to get access if you sign up via the changelog here https://github.blog/changelog/2024-10-01-evolving-github-issues-public-preview/

you should get added within 24-48 hours. 

If you have a request above which hasn't been added yet please just sign up and then you will get added 🎉

**Reply by nholthaus** [reply]
please add https://github.com/nholthaus

**Reply by ruca-sential** [reply]
Please add to the projects under https://github.com/orgs/Passential to this beta. The users are: ruca-sential & CS-Passential. Thank you!!

**Reply by stbrody** [reply]
Sub-issues is a great feature and I'm excited for it to roll out.  But IMO that's not the same as relations between issues.  There are lots of different types of relationships I may want to express between issues beyond just parent/child relationships. For example: is blocking / is blocked by, duplicate of / is duplicated by, is related to, documents / is documented by, tests / is tested by, etc

**Reply by prossm** [reply]
Thank you @azenMatt The team at Idealist.org is interested!

https://github.com/orgs/idealist7

**Reply by toddjonker** [reply]
Sub-issues are effectively the opposite of blocking relationships: a sub-issue is sensibly assumed to be required _before_ this task can be completed, but a blocked-issue is something that cannot happen until _after_ this task is completed.

I just tried to model a blocking issue as subtasks of the blocked issues, but an issue cannot have multiple parents.  So we're left with use subtasks "against the grain" which is not a satisfying long-term solution.

**Reply by azenMatt** [reply]
We have dependencies coming up on the roadmap, I can see that marking this as answered creates confusion that sub-issues considers this 'done' rather than being a first step so i'l go ahead and mark as unanswered :)

**Comment by loeffel-io** [comment]
hey @azenMatt, thanks for inviting! there is an bug i think

<img width="940" alt="Bildschirmfoto 2024-09-11 um 16 19 47" src="https://github.com/user-attachments/assets/a103bafd-07b1-4d80-a082-8e8850a90192">

**Reply by loeffel-io** [reply]
now it works somehow, but it would be really great so see the hierarchy in the board 

<img width="376" alt="Bildschirmfoto 2024-09-11 um 16 24 14" src="https://github.com/user-attachments/assets/d1827a77-791a-4d75-872b-4a12273b38da">

**Reply by loeffel-io** [reply]
a view like this would be a game changer btw

https://trello.com/power-ups/6333f76a988ceb01e68c006e/dependencies-graph

**Reply by azenMatt** [reply]
you should be able to customise what you can see on the board in the views 'field' menu:

![image](https://github.com/user-attachments/assets/5a38746d-1f2a-447a-80c9-40d3fb94f0c8)

here you can pick to see the parent issue. Or the roll up for progress

**Reply by jvani** [reply]
@azenMatt as a related feature request, if we story point issues using a GH Project field it would be useful to see progress by story points of child issues.

**Comment by maliberty** [comment]
@azenMatt what is the timeline to release to production?  I'm trying to see if I want to beta or just wait.  Thanks.

**Reply by azenMatt** [reply]
not long, a few weeks maybe

**Comment by azenMatt** [comment]
Hi all - this is now in public preview! https://github.blog/changelog/2024-10-01-evolving-github-issues-public-preview/

Please sign up using that link if you would like access :)

**Reply by wodow** [reply]
OP's original question is about issues blocking issues, but "block" doesn't appear in that blog post. Can we use the new issues features to record one issue blocking another?

**Reply by justyn** [reply]
Yes @azenMatt this discussion was about blocking issues - what is the status of that? Is that abandoned in favour only of this looser arrangement (one issue is a sub-issue of another)

**Comment by scottconnerly** [comment]
https://github.com/github/roadmap/issues/956 looks like it got pushed back to 2025 Q2. Is that accurate @azenMatt ?

**Comment by demisx** [comment]
@azenMatt Can we please 🙏🏻 get an update on this feature?

**Comment by AleixMT** [comment]
At least I would like to know how much more I will have to wait for this feature to come to official GitHub

**Comment by stbrody** [comment]
I made this comment on a different post in this thread (https://github.com/orgs/community/discussions/4928#discussioncomment-11353058), but re-posting here for visibility:

Sub-issues is a great feature and I'm excited for it to roll out. But IMHO that's not the same as relations between issues. There are lots of different types of relationships I may want to express between issues beyond just parent/child relationships. For example: is blocking / is blocked by, duplicate of / is duplicated by, is related to, documents / is documented by, tests / is tested by, etc

I think this issue has been closed incorrectly.  I do not believe the linked feature of sub-issues actually addresses the request that this post represents.  I believe this issue should be re-opened to capture the fact that there is still an outstanding feature request here that has not been addressed.

**Reply by AleixMT** [reply]
Totally agree. Sub-issues do not address this feature request.

**Reply by stbrody** [reply]
@azenMatt @samus-aran

**Reply by lucianomlima** [reply]
You're right.

**Reply by Norbiros** [reply]
I've noticed a limitation with sub-issues: each sub-issue can only have one parent. This restricts our ability to properly represent relationships like is blocking or is blocked by.

**Comment by AleixMT** [comment]
@azenMatt how much time until it's in production? Thank you

**Comment by sdepouw** [comment]
Chiming in here: "Sub-issues" is indeed insufficient. Compare with Azure DevOps.

You can't even use sub-issues for this purpose. Say you have "issue 5", which blocks "issue 6" and "issue 7". You might say "set issue 5 as a sub-issue for 6 and 7", but GitHub doesn't allow this (you can set it to 6 *or* 7, but not both).

**Reply by Blackjacx** [reply]
We have exactly the same issue. Our Backlog is in a dedicated repo while iOS/Android teams work on their own repos. Now each team of course wants their epic tickets which group sub issues but the product owners, that manage the backlog also would like to use the new sub issues feature[^1] to visualize progress on the kanban board.

But both is not possible 😔

[^1]: Kudos to the GH team btw. this looks really nice 👍

**Comment by Thiago-Morais** [comment]
Sub-issues is a "1 to many" relation, but many relations require a "many to many" relation like dependency/blocking

Example: 
`Issue 1` is blocked by `Issue 2` and `Issue 3` which both are blocked by `Issue 4`

You can't do this with Sub-Issues

**Comment by azenMatt** [comment]
A quick update on this, we are working on dependencies as the next relationship type following our work on sub-issues and expect to have something out with you all soon!

**Reply by prossm** [reply]
Thank you! We already made the switch to Shortcut (since Pivotal Tracker is no more as of 4/30). But it will be cool to check this out for future projects.

**Reply by s3rg1o7** [reply]
is there an expected release time for the "is blocked by" relationship?

**Reply by stbrody** [reply]
I'm surprised by the decision to roll out each new relationship type independently, vs having a single generic relationship feature where users can add custom labels for the type of the relationship?  Similar to what Jira does.  That would enable addressing all kinds of relationships (depends-on, blocked-by, related-to, documented-by, tested-by, etc) all at once.

**Reply by grafana-dee** [reply]
will happily provide testing of this functionality if needed... or at least happy to work with a PM on it.

the hope is for something that maps a bi-directional relationship between GitHub issues across repos (ideally across orgs, but within an org is good).

inspiration should be:
* JIRA https://confluence.atlassian.com/adminjiraserver/configuring-issue-linking-938847862.html (solid implementation)
* Gitlab https://gitlab.com/gitlab-org/gitlab/-/blob/c03c1be71de6f26f19862ee203084717dbf4c792/doc/user/project/issues/related_issues.md#add-a-linked-issue (solid implementation)
* Linear https://linear.app/docs/issue-relations (best UX)
* MS Project https://support.microsoft.com/en-gb/office/link-tasks-in-a-project-31b918ce-4b71-475c-9d6b-0ee501b4be57 (most mature capability)

ideally we could:
* define relationships as bi-directional things, i.e. "blocks", "is blocked by"
* see the relationships against an Issue
* view all relationships in a repo for filtered Issues (open, etc)
* view all relationships in a GitHub Project based on a filter

**Reply by evi-liu** [reply]
The public preview for issue dependencies is now available 🎉 ! 

If your organization would like to join the preview, just drop your GitHub org name in this discussion thread:
👉 https://github.com/orgs/community/discussions/165749

**Comment by evi-liu** [comment]
Hi all, good news! We now have a **public preview** available for marking issues as “blocked by” and “is blocking”  🎉 !

If your organization would like to join the preview, just drop your GitHub org name in this discussion thread:
👉 https://github.com/orgs/community/discussions/165749

**Reply by prossm** [reply]
Amazing! Thank you, Evi!

**Reply by Norbiros** [reply]
Yayy!

**Reply by sdepouw** [reply]
Nice progress! Hopefully more relationships are made available as well. (Azure DevOps sports a lot, including a generic, but useful "relates to").

**Reply by theoschafer** [reply]
We would love that! org name: Tetsuwan-Scientific

**Comment by evi-liu** [comment, marked as answer]
Hi all, I'm excited to announce that dependencies on issues are now generally available! 🎉 🥳 Give it a try!

https://github.blog/changelog/2025-08-21-dependencies-on-issues/

**Reply by sdepouw** [reply]
Congratulations on deployment! We're already using it for one of our projects (that couldn't participate in the beta), and I have some feedback:

Now that an Issue can be "blocked" (having an `is:blocked` feature and all), it would be nice to be able to manually mark an Issue as Blocked. The reason is, sometimes an Issue can be blocked for reasons other than "it's dependent on another Issue." It's understandable that these Issues would never automatically become unblocked (because there are no associated Issues for them), but in order to fully use this new Blocked structure, it's a necessity. The current workaround is to create "stub" Issues for these blocking points, but it really isn't ideal to have to do that, of course.

With this new relationship capability in place, we hope to see this extend to generic relationship. It doesn't need any automations, but sometimes it's good to be able to say "Isuses 1 and 2 relate to each other" generically. (Azure DevOps supports this, along with a host of other relationships.)

Hopefully we'll continue to see iteration!

**Reply by sophiexeon** [reply]
thank you! perfect timing <3

**Reply by Jakob-Unfried** [reply]
Nice!

Is there any reason why issues cant be "block by" an open PR?

Usually, if I start working on something right away, i only open a PR without a companion issue.
I would love to be able to set other issues as dependent on that new feature being merged.

**Reply by hfhbd** [reply]
Similar, how can I mark a PR as blocked by an issue?

**Comment by naveedhassan1328-hue** [comment]
The “add relation between issues” request is now shipped as Issue dependencies (Blocks / Blocked by). You can use it to model exactly “A blocks B” and “B is blocked by A.”

How to use (UI)

Open an issue → right sidebar → Dependencies (or Add dependency).
Search/select the other issue.
Choose relation: Blocks or Blocked by.
The relationship is shown on both issues and updates subscribers.
Notes and tips

Works across repositories you can access; both sides must be visible to the viewer.
Requires at least Triage (or higher) permission to add/remove dependencies.
Task lists still create Tracked by relations (parent ↔ child) and are separate from dependencies.
Linked pull requests remain separate; dependencies are for issue↔issue only.
If you don’t see Dependencies, check repo permissions or settings; some orgs may roll it out gradually.
Automation ideas

Use a label like blocked and a GitHub Action to prevent merging PRs that close an issue while its blockers are open (by querying issues and failing a status check). Dependencies don’t automatically gate merges—use Actions/branch protection for that policy.

**Comment by cam-radical** [comment]
would also love to see generic "relates to"

**Reply by tickytacky** [reply]
YES, this is also something that we need as well, as we are on our way from gitlab to github. We've used that "relates to" feature quite excessively in gitlab. Don't know how the github users can live without it...!?! This is an important feature for structured, comprehensible and efficient documentation of Issues even accross Repositories.

**Reply by swefd** [reply]
This is the reason I'm here again :D We definitely need this.
Would love to see this. Would be even better to have an option to create your own relation types in the future.

**Reply by sunnyi101** [reply]
We’re looking for users to test a new Relates to relationship for issues. It lets you connect related issues without implying that one blocks, duplicates, or contains another.

If you’re interested, reply with your GitHub username and the repositories where you’d like to try it. We’d love to opt you in and hear your feedback!

**Reply by hfhbd** [reply]
@sunnyi101 I would like to test it with my org: [softwork-app](https://github.com/softwork-app). Any plans to support relationships with Pull Requests too, mostly `blocked by`?

**Reply by tickytacky** [reply]
Would love to participate but unfortunately I'm on vacation the next four weeks... :-(

**Reply by Norbiros** [reply]
@Norbiros If you can enable it for organization: https://github.com/Zerya-Dev/, if only for a specific repo: https://github.com/Hack4Krak/Hack4KrakSite/. Thanks!

**Reply by agilesteel** [reply]
Please enable for https://github.com/formslogic/fl 🙏🏼

**Reply by tickytacky** [reply]
Would it also be possible to enable Repos in GHE to test the feature?

**Reply by swefd** [reply]
@sunnyi101 Yes, please enable it for https://github.com/webspirio/travel-crm and https://github.com/webspirio/ua-well-portal

Thank you

**Reply by sunnyi101** [reply]
I enabled it for you all. Please try it out and let me know if you find any issues!

**Reply by davesmits** [reply]
Would love to have it in nxtsoftnl organization

**Reply by ptsouchlos** [reply]
@sunnyi101 Please enable this for @xstrahl , thank you!

**Reply by AleixMT** [reply]
@sunnyi101 Apply it into github.com/ASBTEC and github.com/ICIQ-DMP. Thank you so much!

**Reply by sunnyi101** [reply]
We've shipped this globally to everyone! Please check it out:
https://github.blog/changelog/2026-08-07-connecting-issues-and-multi-select-field-support/

**Comment by aaravrav** [comment]
Being able to block open PRs on other open PRs would be massively useful as well
